File: ehr/test/src/org/labkey/test/components/ehr/panel/AnimalSearchPanel.java
package org.labkey.test.components.ehr.panel;

import org.labkey.test.components.ext4.widgets.SearchPanel;
import org.openqa.selenium.WebDriver;

public class AnimalSearchPanel extends SearchPanel
{
    public AnimalSearchPanel(WebDriver driver)
    {
        this(DEFAULT_TITLE, driver);
    }

    public AnimalSearchPanel(String title, WebDriver driver)
    {
        super(title, "ehr-animalsearchpanel-", driver);
    }
}

File: ehr/resources/web/ehr/panel/AnimalSearchPanel.js
/*
 * Copyright (c) 2017 LabKey Corporation
 *
 * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
 */

/**
 * Constructs a new EHR AnimalSearchPanel
 * @class
 * @cfg filterTypes
 * @cfg reports
 *
 */
Ext4.define('EHR.panel.AnimalSearchPanel', {
    extend: 'LABKEY.ext4.SearchPanel',
    alias: 'widget.ehr-animalsearchpanel',
    schemaName: 'study',
    queryName: 'Demographics',
    viewName: 'Search Panel',
    defaultViewName: 'Alive, at Center',
    title: 'Search Criteria',
    LABEL_WIDTH: 200,
    width: 660
});
